server/mailer/index.js

```diff
@@ -67,10 +72,12 @@ const sendSignUpConfirmationLink = (req, resp) => {
 const sendResetPasswordLink = (req, resp) => {
   const { email: receiver, displayName, resetToken } = resp.locales;
   const host = req.get('host');
+  const hours = getHours(EXPIRATION_TIME);
   const resetUrl = `${host}/auth/recovery-password/${resetToken}`;
   const title = `${PROJECT_NAME} - Reset your password`;
-  const info =
-    'Reset your password by clicking reset button. If you have not requested password reset to your account, just ignore this message.';
+  const info = `<p>Reset your password by clicking reset button. If you have not requested password reset to your account, just ignore this message.</p><p><b>Remember that reset button will be active only for ${
```

Comment:
Please extract expiration message as a separate string and then concatenate it to compose the whole message.

Reply:
Please investigate how to handle the expiration message:
- Should it really be bold (important for a user to read)?
- Is the placement correct? Consider moving it to the footer.

Reply:
> Please investigate how to handle the expiration message:
> 
> * Should it really be bold (important for a user to read)?
> * Is the placement correct? Consider moving it to the footer.

Generally from what I've found results that most of the services does not provide info about link expiration in their emails. But this was our customer request. 
I will put this as normal font-weight. Not bolded and we should be good to go